src/utils/createMusicMetadata.ts

```diff
@@ -33,7 +35,7 @@ const music_metadata = {
 	},
 	isrc: '', // CC-XXX-YY-NNNNN
 	artwork: {
-		uri: '', // ipfs://bafkreibjsjxxys2pl6wijorije7yixjsw7xlho5krqdimwayyobnybewey
+		uri: undefined, // ipfs://bafkreibjsjxxys2pl6wijorije7yixjsw7xlho5krqdimwayyobnybewey
```

Comment:
again, uri MUST be defined as a string to meet music metadata standards

https://gist.github.com/bretth18/df8358c840fa94946ec212f753e290dd

Reply:
yeah I understand that, it's because the typing of NFT.Storage. 

When you are sending in formData to the metadata it's not a string but gets turned into a string when it's returned from NFT.storage's call. 

`await client.store(metadata)`

https://nftstorage.github.io/nft.storage/client/classes/lib.NFTStorage.html#store see the requirements for this method. 


This is a NFT.Storage specific problem.

We are typecasting a File type to string before submitting to NFT.storage which causes the error.